Tests/NIOTests/IOErrorTest+XCTest.swift

```diff
@@ -0,0 +1,35 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
```

Comment:
@realdoug sorry, tiny nit, can you put 2020 in the copyright instead of 2017-2018 😬

Reply:
did you mean to set this one back to 2017-2018?  That was my fault for setting IO.swift, should have been just the new files IIUC.  I'll push the correct update if so.

Reply:
@realdoug yeah, sorry, the autogenerated files need to remain as they are. Our CI gets unhappy otherwise (it checks the files aren’t edited). I know I commented (by accident) on the auto generated file :). Sorry about that